.github/workflows/cdk_connectors_tests.yml

```diff
@@ -54,11 +53,11 @@ jobs:
             }' \
 
   connectors_ci:
-    needs: changes
+    needs: cdk_changes
     # We only run the Connectors CI job if there are changes to the connectors on a non-forked PR
     # Forked PRs are handled by the community_ci.yml workflow
     # If the condition is not met the job will be skipped (it will not fail)
-    if: github.event_name == 'pull_request' && needs.changes.outputs.python_cdk == 'true' && github.event.pull_request.head.repo.fork != true
+    if: github.event_name == 'pull_request' && needs.cdk_changes.outputs.python_cdk == 'true' && github.event.pull_request.head.repo.fork != true
```

Comment:
You still have `workflow_dispatch` trigger. It's usually handy to keep it in order to manually run the workflow. Your `if` here will not run the `connectors_ci` job on workflow dispatch...

Reply:
Checking this error, I was checking the other workflows and it feels like [this](https://github.com/airbytehq/airbyte/blob/4e9146425b00d98b9d144d71def7e2496251d3de/.github/workflows/connectors_tests.yml#L87-L90) is not used. I provided a fix for that too